.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+
+  # Maintain dependencies for Docker Images
+  - package-ecosystem: "docker"
```

Comment:
I'm curious what this will find; in general we're pretty intentional about bumping things in the Dockerfile.

Reply:
It will generate PRs for all of them. Which lines should we ignore?

Reply:
Probably should ignore these, as the golang and kubernetes versions need to be manually kept in sync, and only bumped as part of the patch release process
* https://github.com/rancher/rke2/blob/master/Dockerfile#L4
* https://github.com/rancher/rke2/blob/9ce3825ec005872a0873307844ba99a2bdd4ee19/Dockerfile#L141
* https://github.com/rancher/rke2/blob/9ce3825ec005872a0873307844ba99a2bdd4ee19/Dockerfile#L143

Reply:
I can [configure dependabot to ignore certain dependencies](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#ignore)

Reply:
these new lines should have dependabot completely ignore those two dependencies in all Dockerfiles, as PRs are generated we can iterate on this and add more lines to ignore if we chose

Reply:
Just a note/reminder that Updatecli can be used for such updates in case more control logic is needed around which images to update and how to bump them.